pkg/cmd/auth/login/login.go

```diff
@@ -159,15 +159,12 @@ func loginRun(opts *LoginOptions) error {
 	existingToken, _ := cfg.AuthToken(hostname)
 	if existingToken != "" && opts.Interactive {
 		if err := shared.HasMinimumScopes(httpClient, hostname, existingToken); err == nil {
-			var keepGoing bool
-			err = prompt.SurveyAskOne(&survey.Confirm{
-				Message: fmt.Sprintf(
-					"You're already logged into %s. Do you want to re-authenticate?",
-					hostname),
+			keepGoing, err := opts.Prompter.Confirm(cmdutil.ConfirmOpts{
```

Comment:
Would there be value in changing the signature of Prompter methods to only take primitives as arguments and avoid having to import a separate package just for the caller to supply their options via struct?

For example:
- `Confirm(message string, defaultValue bool)`
- `Input(message string, defaultValue string)`
- `Select(message string, options []string)` 

The goal would be to avoid exposing all possible Survey features since most of them we don't use. Then, when we need a different kind of prompt, for example one with a specific validator, we would define a separate method for it:
- `InputHostname(message string, defaultValue string)` - like `Input()` but applies a hostname validator